annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -172,16 +172,11 @@ public AnnotatedRecord annotateRecord(MutationRecord record, boolean replaceHugo
                 additionalProperties);
         }
 
-        // first, get the mutation in the right notation
-        String hgvsNotation = convertToHgvs(record);
-
         // make the rest call to genome nexus
         RestTemplate restTemplate = new RestTemplate();
         HttpEntity<LinkedMultiValueMap<String, Object>> requestEntity = getRequestEntity();
 
-        log.debug("Annotating: " + hgvsNotation + " from sample " + record.getTumor_Sample_Barcode());
-
-        ResponseEntity<GenomeNexusAnnotationResponse[]> responseEntity = restTemplate.exchange(hgvsServiceUrl + hgvsNotation + "?" + isoformQueryParameter + "=" + isoformOverridesSource + "&" + hotspotParameter + "=summary", HttpMethod.GET, requestEntity, GenomeNexusAnnotationResponse[].class);
+        ResponseEntity<GenomeNexusAnnotationResponse[]> responseEntity = restTemplate.exchange(getUrlForRecord(record, isoformOverridesSource), HttpMethod.GET, requestEntity, GenomeNexusAnnotationResponse[].class);
```

Comment:
good change - the inline construction of the URL made it hard to see the arguments to exchange()